pkg/backend/filestate/backend.go

```diff
@@ -153,6 +153,14 @@ func New(d diag.Sink, originalURL string) (Backend, error) {
 		}
 	}
 
+	isAcc, err := bucket.IsAccessible(context.TODO())
+	if err != nil {
+		return nil, fmt.Errorf("unable to access bucket %s: %w", u, err)
```

Comment:
yeh Ians suggestion is a better message, this err is that we couldn't check if the bucket was accessible (like a network timeout) rather than we got a code back saying it couldn't be accessed.

Reply:
👍 I'll change the comment